guess_the_number.py

```diff
@@ -51,10 +51,11 @@ def main():
         guess = get_guess()
         result = check_guess(guess, secret)
         print(result)
-
         if result == correct:
             print(f'It took you {guess_count} tries!')
-            break
+            repeat = input('Would you like to play again? Y or N: ').lower()
```

Comment:
should we also add strip to remove whitespace?

Reply:
Probably a good idea. Do you know, can I make the change and recommit into this same pull request or do I need to create another?